avaframe/out3Plot/outQuickPlot.py

```diff
@@ -71,10 +73,17 @@ def generatePlot(dataDict, avaName, outDir, cfg, plotDict):
     nx_loc = int(ny * 0.5)
 
     # Difference between datasets
-    dataDiff = np.nan_to_num(data1 - data2)
-    diffMax = np.amax(dataDiff)
-    diffMin = np.amin(dataDiff)
-    diffMean = np.mean(dataDiff)
+    dataMask = np.ones((ny, nx))
+    for m in range(ny):
+        for k in range(nx):
+            if data1[m,k] == 0.0 and data2[m,k] == 0.0:
+                dataMask[m,k] = np.nan
+
+    dataDiff = data1 - data2
+    dataDiff[np.isnan(dataMask)] = np.nan
```

Comment:
dataDiff = data1 - data2
dataDiff = np.where((data1==0 & data2==0), np.nan, dataDiff)
should do the same

Reply:
I changed this